static/js/model.js

```diff
@@ -1387,7 +1387,7 @@ Apigee.APIModel.Methods = function() {
     }
     requestContainerString += "</dl>";
     requestContainerElement.html(requestContainerString);
-    curlContainerElement.html("<pre>"+Apigee.curl+"</pre>");
+    curlContainerElement.html("<pre>" + Apigee.curl.replace(/&/, '&amp;').replace(/</, '&lt;').replace(/>/, '&gt;').replace(/"/, '&quot;') + "</pre>");
```

Comment:
Though perhaps not entirely necessary, it seems like it would be stylistically consistent to also escape single quotes.